src/Tracing/TransactionContext.php

```diff
@@ -136,18 +136,42 @@ public static function fromSentryTrace(string $header): self
         return $context;
     }
 
+    /**
+     * Returns a context populated with the data of the given environment variables.
+     * Defaults to SENTRY_TRACE and SENTRY_BAGGAGE if no values are passed in.
+     *
+     * @param string $sentryTrace The sentry-trace value from the environment
+     * @param string $baggage     The baggage header value from the environment
+     */
+    public static function fromEnvironment(?string $sentryTrace = null, ?string $baggage = null): self
+    {
+        if (null === $sentryTrace) {
+            $sentryTrace = getenv('SENTRY_TRACE');
+        }
+        if (null === $baggage) {
+            $baggage = getenv('SENTRY_BAGGAGE');
+        }
```

Comment:
@stayallive We can also eliminate these default values and pass in the env vars explicitly in Laravel/Symfony.